src/DotNet.Status.Web/Models/IssueMentionForwarder.cs

```diff
@@ -0,0 +1,97 @@
+    {
+        public string[] WatchedTeams { get; set; }
+        public string[] IgnoreRepos { get; set; }
+        public string TeamsWebHookUri { get; set; }
+    }
+
+    public interface IIssueMentionForwarder
+    {
+        Task<bool> HandleIssueBody([CanBeNull] string oldBody, string newBody, string title, string commentUri, string username, DateTimeOffset date);
+    }
+
+    public class IssueMentionForwarder : IIssueMentionForwarder
+    {
+        private readonly IOptionsSnapshot<IssueMentionForwardingOptions> _options;
+        private readonly ILogger<IssueMentionForwarder> _logger;
+        private readonly IHttpClientFactory _httpClientFactory;
+
+        public IssueMentionForwarder(IHttpClientFactory httpClientFactory, IOptionsSnapshot<IssueMentionForwardingOptions> options, ILogger<IssueMentionForwarder> logger)
+        {
+            _httpClientFactory = httpClientFactory;
+            _options = options;
+            _logger = logger;
+        }
+
+        public async Task<bool> HandleIssueBody([CanBeNull] string oldBody, string newBody, string title, string commentUri, string username, DateTimeOffset date)
+        {
+            var teams = _options.Value.WatchedTeams;
+            var foundTeam = teams.FirstOrDefault(newBody.Contains);
+            if (string.IsNullOrEmpty(foundTeam))
+            {
+                return false;
+            }
+            if (!string.IsNullOrEmpty(oldBody))
+            {
+                var oldTeam = teams.FirstOrDefault(oldBody.Contains);
+                if (oldTeam == foundTeam)
+                {
+                    return false;
+                }
+            }
+
+            await SendCommentNotification(title, newBody, foundTeam, commentUri, username, date);
+            return true;
+        }
+
+        private async Task SendCommentNotification(string title, string body, string teamName, string commentUri, string username, DateTimeOffset date)
+        {
+            var message = new MessageCard
+            {
+                ThemeColor = "0072C6",
+                Text = $"Team {teamName} was mentioned in an issue.",
+                Actions = new List<IAction> {
+                    new OpenUri {
+                        Name = "Open comment",
+                        Targets = new List<Target>
+                        {
+                            new Target
+                            {
+                                OperatingSystem = "default",
+                                Uri = commentUri
+                            }
+                        }
+                    }
+                },
+                Sections = new List<Section> {
+                    new Section {
+                        ActivityTitle = title,
+                        ActivitySubtitle = $"{username} on {date:g}",
+                        ActivityText = body
+                    }
+                }
+            };
+            HttpClient client = _httpClientFactory.CreateClient();
```

Comment:
Do we not need to "using" thing?  We created a thing that is IDisposable.

Reply:
We do. I think the HttpClientFactory stuff does the right thing, but I will stick a using here.